server/routes/temporary-accommodation/index.ts

```diff
@@ -20,5 +22,9 @@ export default function routes(controllers: Controllers, services: Services): Ro
 
   manageRoutes(controllers, services, router)
 
+  if (config.environment === 'local' || config.environment === 'dev') {
```

Comment:
It might be an idea to allow this to work on our 'test' environment too as that is somewhere we might like to do user testing with this 👍

Reply:
Cool, changed